aviary/mission/gasp_based/ode/taxi_eom.py

```diff
@@ -52,6 +62,8 @@ def compute(self, inputs, outputs):
         outputs[Dynamic.Mission.MASS] = takeoff_mass - outputs["taxi_fuel_consumed"]
```

Comment:
I think we should abandon the vectorization of the taxi_eom, since we aren't really discretizing it. We really just have a starting mass, a delta_t, and a fuelflow rate.  All we really compute is the ending mass, which is start mass - fuelflow*delta_t. With the new changes, only fuelflow is a vector. To make this work, delta_t in this calculation would need to be an array of time points (something like np.linspace(0, 1, num_nodes) * dt_taxi instead of dt_taxi.) It just doesn't seem worth it, because we aren't doing a meaningful calculation here,  Fuelflow is always assumed constant over the taxi.  If it ever becomes time-varying (i.e., we are letting the optimizer pick taxi throttle speeds), then it might become relevant.

Reply:
done.